nvm.sh

```diff
@@ -1303,6 +1320,30 @@ nvm_die_on_prefix() {
   fi
 }
 
+# Succeeds if $IOJS_VERSION represents an io.js version that has a
+# Solaris binary, fails otherwise.
+# Currently, only io.js 3.3.1 has a Solaris binary available, and it's the
+# latest io.js version available. The expectation is that any potential io.js
+# version later than v3.3.1 will also have Solaris binaries.
+iojs_version_has_solaris_binary() {
+  local IOJS_VERSION=$1
```

Comment:
Updated this PR with another commit that does that. Let me know if that's not what you had in mind.